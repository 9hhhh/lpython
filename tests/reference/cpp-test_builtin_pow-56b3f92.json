{
    "basename": "cpp-test_builtin_pow-56b3f92",
    "cmd": "lpython --no-color --show-cpp {infile}",
    "infile": "tests/../integration_tests/test_builtin_pow.py",
    "infile_hash": "5696eb31b14c38dcd0c4d04d12af41c66c8efd457a54b9adbce00229",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "cpp-test_builtin_pow-56b3f92.stderr",
    "stderr_hash": "d20e661f54ebada95c60e47323001f6f5d147c5731aac8251027a03a",
    "returncode": 1
}