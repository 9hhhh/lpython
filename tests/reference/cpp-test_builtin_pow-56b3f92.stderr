style suggestion: Could have used '**' instead of 'pow'
  --> tests/../integration_tests/test_builtin_pow.py:11:12
   |
11 |     assert pow(a, b) == 32
   |            ^^^^^^^^^ '**' could be used instead
Internal Compiler Error: Unhandled exception
Traceback (most recent call last):
[2m  File "[0m[1m[35m$DIR/src/bin/lpython.cpp[39m[0m[2m", line 831[0m
    return emit_cpp(arg_file, runtime_library_dir, compiler_options);
[2m  File "[0m[1m[35m$DIR/src/bin/lpython.cpp[39m[0m[2m", line 217[0m
    auto res = LFortran::asr_to_cpp(al, *asr, diagnostics);
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_cpp.cpp[39m[0m[2m", line 475[0m
    v.visit_asr((ASR::asr_t &)asr);
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3108[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3086[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3109[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 2911[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_cpp.cpp[39m[0m[2m", line 205[0m
    ASR::symbol_t *mod = x.m_global_scope->get_symbol(item);
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3111[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 2920[0m
[2m  File "[0m[1m[35m../libasr/codegen/asr_to_c_cpp.h[39m[0m[2m", line 171[0m
[2m  File "[0m[1m[35m../libasr/codegen/asr_to_c_cpp.h[39m[0m[2m", line 373[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3125[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 2953[0m
[2m  File "[0m[1m[35m../libasr/codegen/asr_to_c_cpp.h[39m[0m[2m", line 978[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3166[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 2987[0m
[2m  File "[0m[1m[35m../libasr/codegen/asr_to_c_cpp.h[39m[0m[2m", line 717[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3166[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3001[0m
[2m  File "[0m[1m[35m../libasr/asr.h[39m[0m[2m", line 3182[0m
LFortranException: visit_LogicalNot() not implemented
